drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -737,30 +760,28 @@ static int bq274xx_exit_shutdown_mode(const struct device *dev)
 	return 0;
 }
 
-static int bq274xx_pm_action(const struct device *dev,
+static int pm_action(const struct device *dev,
 			     enum pm_device_action action)
 {
-	int ret;
-
 	switch (action) {
 	case PM_DEVICE_ACTION_TURN_OFF:
-		ret = bq274xx_enter_shutdown_mode(dev);
+		status = enter_shutdown_mode(dev);
```

Comment:
Do you mind if we use `ret` instead of `status`? `ret` is much more widely used in the Zephyr code base.

```suggestion
		ret = enter_shutdown_mode(dev);
```